modules/javafx.controls/src/main/java/javafx/scene/control/TreeTableCell.java

```diff
@@ -712,39 +712,42 @@ private void updateItem(int oldIndex) {
      **************************************************************************/
 
     /**
-     * Updates the TreeTableView associated with this TreeTableCell. This is typically
-     * only done once when the TreeTableCell is first added to the TreeTableView.
-     *
+     * Updates the {@code TreeTableView} associated with this {@code TreeTableCell}.
+     * This is typically only done once when the {@code TreeTableCell} is first
+     * added to the {@code TreeTableView}.
+     * <p>
      * Note: This function is intended to be used by experts, primarily
      *       by those implementing new Skins. It is not common
      *       for developers or designers to access this function directly.
-     * @param tv the TreeTableView associated with this TreeTableCell
+     * @param tv the {@code TreeTableView} associated with this {@code TreeTableCell}
      */
     public final void updateTreeTableView(TreeTableView<S> tv) {
```

Comment:
The changes to the docs for this method were just formatting changes to match the corresponding changes to row and column methods (I did the same thing earlier for the property methods).